srcpkgs/NetworkManager/patches/fix-cross-gir.patch

```diff
@@ -1,52 +1,29 @@
 Reason: Fix crossbuilding GIR
 Upstream: No, not upstreamable since this is very specific to the Void setup
```

Comment:
This patch is needed when `-Ddocs=true`, either remove it or enable documentation.
I think it's better to have docs on.
Note that `-Ddocs=true` depends on `-Dintrospection=true`.

Reply:
Does `$(vopt_bool gir introspection)` in your patch below function the same as adding `-Dintrospection=true`?

Reply:
Yes, when `build_option_gir` on.